resizeobserver/demo.js

```diff
@@ -0,0 +1,13 @@
+/* global ResizeObserver */
+
+const ro = new ResizeObserver(entries => {
```

Comment:
I can’t quite follow. Changing the background pattern is just as much a (useless) visual change as changing the element to a circle. I am trying to show _how_ to use RO, which this easily conveys (and can be tested by the user). What am I missing?